Source/WTF/wtf/ThreadAssertions.h

```diff
@@ -25,13 +25,22 @@
 
 #pragma once
 
-#include <wtf/Compiler.h>
-#include <wtf/MainThread.h>
 #include <wtf/ThreadSafetyAnalysis.h>
-#include <wtf/Threading.h>
 
 namespace WTF {
 
+class ThreadlikeAssertion;
+
+class Threadlike {
```

Comment:
I think "thread-like" is two words so I would have chosen "ThreadLike" instead of "Threadlike".

Reply:
done